### DISCUSSION TITLE
Virtual level Markers for a level target range in OSRS

### DISCUSSION THREAD

**Discussion by CeeGeeZ** [runelite/runelite#16777] [Ideas]
![Virtual levels](https://github.com/runelite/runelite/assets/139237186/ede0c396-80a6-4364-b7e7-f4d4557e760f)

This simple quality of life plugin will add markers across the experience bar, for visualising level milestones throughout an xp range. So in theory, should a new player set a level range from 1 to 99, we would see 92 markers up to the halfway point of the experience bar, and 7 markers over the second half. This could be a really great way to visualise milestones for long grinds in OSRS.

**Comment by Alexsuperfly** [comment]
the xp tracker plugin does this, if you want its display in the game window you can right click and "add to canvas" it.
